package.json

```diff
@@ -16,8 +16,7 @@
   ],
   "author": "",
   "license": "MIT",
-  "repository": {
-  },
+  "repository": {},
```

Comment:
Reverted this change. Will fill in all the package information in a subsequent PR (not planning to release this until we finish some housekeeping)